packages/dashboard/src/components/DashboardFrame/Navigation.tsx

```diff
@@ -1,62 +1,73 @@
 import {
     List,
-    ListItem,
+    ListItem as MuiListItem,
     Box,
     ListItemText,
     ListItemIcon,
     Collapse,
-    makeStyles,
     Theme,
     ListItemProps,
+    useMediaQuery,
+    experimentalStyled as styled,
+    BoxProps,
 } from '@material-ui/core'
 import { Masks, AccountBalanceWallet, ExpandLess, ExpandMore, Settings } from '@material-ui/icons'
-import { useState } from 'react'
+import { useContext } from 'react'
 import { useRouteMatch } from 'react-router'
 import { Link, LinkProps } from 'react-router-dom'
 import { Routes } from '../../pages/routes'
+import { DashboardContext } from './context'
+import Logo from './Logo'
 
-const useStyle = makeStyles((theme: Theme) => ({
-    selected: {
+const ListItemLink = styled(({ nested, ...props }: LinkProps & ListItemProps & { nested?: boolean; to: string }) => (
+    <MuiListItem button component={Link} selected={!!useRouteMatch(props.to)} {...props} />
+))(({ theme, nested }) => ({
+    '&.MuiListItem-root': {
```

Comment:
I'm sorry maybe I'm not clear on my previous review. We should avoid specifying the global Material UI class names for now (in this case, `MuiListItem-root`). I have drafted my reason https://github.com/mui-org/material-ui/issues/20012#issuecomment-769642156 and requesting suggestions from the library team in the link.

Reply:
You can keep this style of code temporally as-is. I'll wait for the team to respond.